src/content/docs/apm/agents/ruby-agent/api-guides/ruby-ai-monitoring-apis.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: AI monitoring APIs
```

Comment:
Realizing this is already in the API section, maybe we can rename to just AI Monitoring?

```suggestion
title: AI Monitoring
```